packages/lib/src/components/helpers/QRLoaderContainer.tsx

```diff
@@ -27,14 +28,15 @@ export interface QRLoaderContainerProps extends UIElementProps {
 }
 
 class QRLoaderContainer<T extends QRLoaderContainerProps = QRLoaderContainerProps> extends UIElement<T> {
-    // Using the generic here allow to fully extend the QRLoaderContainer (including it's props)
+    // Using the generic here allow to fully extend the QRLoaderContainer (including its props)
     protected static defaultProps = {
         qrCodeImage: '',
         amount: null,
         paymentData: null,
         onError: () => {},
         onComplete: () => {},
-        onActionHandled: () => {}
+        onActionHandled: () => {},
+        showPayButton: false
```

Comment:
Why `showPayButton: false` here?

Reply:
QRCode components, outside of the Dropin, don't need to show a pay button

Reply:
But we want the showPayButton to default to `true`, right? In this case, it is defaulting to false.

I tested on dev playground, and if I create the Swish component ( `checkout.create('swish')...`). , the Pay button isn't displayed with this change. Removing the default prop make the button show up.

Reply:
Ok, I'll set it to true!